### PR TITLE
Send timeserieses to stackdriver in batches.

### PR DESCRIPTION
**Pull request by loburm**
Stackdriver has a limit of 200 timeserieses per request. To be able to
send more we need to send them in batches.

### CODE REVIEW

File: prometheus-to-sd/translator/stackdriver.go [hunk 1]
```diff
@@ -18,27 +18,49 @@ package translator
...
-	req := &v3.CreateTimeSeriesRequest{TimeSeries: ts}
 	proj := fmt.Sprintf("projects/%s", config.Project)
 
-	_, err := service.Projects.TimeSeries.Create(proj, req).Do()
-	if err != nil {
-		glog.Errorf("Error while sending request to Stackdriver %v", err)
-	} else {
-		glog.V(4).Infof("Successfully sent %v timeserieses to Stackdriver", len(req.TimeSeries))
+	var wg sync.WaitGroup
+	wg.Add(numberOfBatches(len(ts)))
```
Comment (piosz) [inline prometheus-to-sd/translator/stackdriver.go]: You don't need this function: you can just have `wg.Add(1)` right before `go func(...)` below. It'd be less error prone.

Comment (loburm) [inline prometheus-to-sd/translator/stackdriver.go]: Thanks, I have updated the code.

**Pull request merged**